src/sonic-ctrmgrd/tests/kube_commands_test.py

```diff
@@ -13,6 +13,8 @@
 
 
 KUBE_ADMIN_CONF = "/tmp/kube_admin.conf"
+FLANNEL_CONF_FILE = "/tmp/flannel.conf"
+CNI_DIR = "/tmp/cni/net.d"
```

Comment:
Why not /etc/cni/net.d?

Reply:
This is test code, where there is NO kubernetes installed. All kubernetes calls are mocked and dummy. The test code does not have privilege to operate on /etc, so we set it as /tmp